scripts/label_validator.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # must be set if you want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+# From Github API Docs why using API for Issue.
+#   You can use the REST API to create comments on issues and pull requests. Every pull request is an issue, but not every issue is a pull request.
+
+EVENT=$1 
+PARAM=$2
+
+function runOnRealase() {
+  latest=$(curl -H "X-GitHub-Api-Version: 2022-11-28" \
+                -sS "https://api.github.com/repos/kyma-project/btp-manager/releases/latest" | 
+                jq -r '.tag_name')
+  notValidPrs=()
+  while read -r commit; do
+    pr_id=$(curl -L \
+              -H "Accept: application/vnd.github+json" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              "https://api.github.com/search/issues?q=$commit+repo:kyma-project/btp-manager+type:pr" |
+              jq 'if (.items | length) == 1 then .items[0].number else empty end')
+
+    if [[ -z $pr_id ]]; then
+      echo "not found PR number for given commit $commit"
+      exit 1
```

Comment:
Does it mean that we exit the workflow if we cannot find PR for given commit hash? Could it block the release?